### PR TITLE
Create jobs browse component

### PR DESCRIPTION
**Pull request by idreturned1**

### CODE REVIEW

File: src/app/modules/dashboard/modules/jobs/components/browse/browse.component.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@import "../../jobs.component.scss";
+
+.jobs-browse-container {
+    display: flex;
+    overflow: hidden;
+    margin: auto;
+    height: calc(100vh - 100px);
+    .job-list-container {
+        display: flex;
+        flex-direction: column;
```
Comment (vivsvaan) [inline src/app/modules/dashboard/modules/jobs/components/browse/browse.component.scss:10]: use flex-col class

File: src/app/modules/dashboard/modules/jobs/components/browse/components/browse-description/browse-description.component.scss [hunk 2]
```diff
@@ -0,0 +1,163 @@
+@import "../../browse.component.scss";
+
+.jobs-description-container {
+    padding: 16px 12px;
```
Comment (vivsvaan) [inline src/app/modules/dashboard/modules/jobs/components/browse/components/browse-description/browse-description.component.scss:4]: you can use py-16 and px-12

File: src/app/modules/dashboard/modules/jobs/components/browse/components/browse-description/browse-description.component.scss [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+            display: flex;
+            .description-details-box {
+                flex: 1;
+            }
+        }
+    }
+    .jobs-company-container {
+        display: flex;
+        padding: 20px 24px 12px;
+        flex-direction: column;
```
Comment (vivsvaan) [inline src/app/modules/dashboard/modules/jobs/components/browse/components/browse-description/browse-description.component.scss:118]: use flex-col class

File: src/app/modules/dashboard/modules/jobs/components/browse/components/browse-description/browse-description.component.scss [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+                width: 72px;
+                img {
+                    max-width: 100%;
+                    max-height: 100%;
+                }
+            }
+            .jobs-company__information {
+                display: flex;
+                flex: 1 1 auto;
+                flex-direction: column;
```
Comment (vivsvaan) [inline src/app/modules/dashboard/modules/jobs/components/browse/components/browse-description/browse-description.component.scss:137]: use flex-col class

Comment (idreturned1) [reply src/app/modules/dashboard/modules/jobs/components/browse/components/browse-description/browse-description.component.scss]: resolved in latest commit

File: src/app/modules/dashboard/modules/jobs/components/browse/components/browse-description/browse-description.component.scss (hunk 3 above)
Comment (idreturned1) [reply src/app/modules/dashboard/modules/jobs/components/browse/components/browse-description/browse-description.component.scss]: resolved in latest commit

File: src/app/modules/dashboard/modules/jobs/components/browse/components/browse-description/browse-description.component.scss (hunk 2 above)
Comment (idreturned1) [reply src/app/modules/dashboard/modules/jobs/components/browse/components/browse-description/browse-description.component.scss]: resolved in latest commit

File: src/app/modules/dashboard/modules/jobs/components/browse/browse.component.scss (hunk 1 above)
Comment (idreturned1) [reply src/app/modules/dashboard/modules/jobs/components/browse/browse.component.scss]: resolved in latest commit

File: src/app/modules/dashboard/modules/jobs/components/browse/components/browse-list/browse-list.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (vivsvaan) [inline src/app/modules/dashboard/modules/jobs/components/browse/components/browse-list/browse-list.component.spec.ts:1]: format file

File: src/app/modules/dashboard/modules/jobs/components/browse/components/browse-description/browse-description.component.spec.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (vivsvaan) [inline src/app/modules/dashboard/modules/jobs/components/browse/components/browse-description/browse-description.component.spec.ts:1]: format file

File: src/app/modules/dashboard/modules/jobs/components/browse/browse.component.spec.ts [hunk 7]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (vivsvaan) [inline src/app/modules/dashboard/modules/jobs/components/browse/browse.component.spec.ts:1]: formatfile

**Pull request merged**